README.md

```diff
@@ -174,6 +174,15 @@ Additional manual steps performed to collect population and/or epidemiological d
 2. There is no automated script for constructing epidemiological variables (e.g. cumulative confirmed cases) as these were manually collected from various Korean provincial websites. Note that these provinces often report the data in different formats (e.g. pdf attachments, interactive dashboards) and usually do not have English translations. For more details on how we collected the data, please refer to the [Data Acquisition and Processing section in the appendix](https://www.dropbox.com/scl/fi/8djnxhj0wqqbyzg2qhiie/SI.gdoc?dl=0&rlkey=jnjy82ov2km7vc0q1k6190esp). 
 This data is saved in [data/interim/korea/KOR_health.csv](data/interim/korea/KOR_health.csv).
 
+
+##### United States
+1. There is no automated script to collect the US policy data. Manually collected policy data is located [here](data/raw/usa/US_covid_policies_pre_school_closure_data.xlsx)
+2. To incorporate US policy on school closures due to COVID-19 into the raw policy dataset, download the most up-to-date information on school closures [here](https://www.edweek.org/ew/section/multimedia/map-coronavirus-and-school-closures.html) and save to [here](data/raw/usa/coronavirus-and-school-closures.html)
+3. Rscript `US_cleaning_school_district_closures.R`: This script cleans the downloaded school closures data and prepares it to be merged with the rest of the raw US policy data that was manually collected. The clean school closure data is saved [here](data/interim/usa/US_school_closures_clean.csv).
```

Comment:
sorry, we're close here @ekrasovich ! Can you put "Rscript"s for these two instructions inside the "`"? See the other sections for reference

Reply:
I'm just going to merge and then start my own PR with some broader readme changes. Thanks!